kubetest2-gke/deployer/deployer.go

```diff
@@ -242,15 +249,17 @@ func bindFlags(d *deployer) *pflag.FlagSet {
 		"are separated by comma, and the ranges of each subnetwork configuration is separated by space.")
 	flags.StringVar(&d.environment, "environment", "prod", "Container API endpoint to use, one of 'test', 'staging', 'prod', or a custom https:// URL. Defaults to prod if not provided")
 	flags.StringSliceVar(&d.projects, "project", []string{}, "Comma separated list of GCP Project(s) to use for creating the cluster.")
-	flags.StringVar(&d.region, "region", "", "For use with gcloud commands to specify the cluster region.")
-	flags.StringVar(&d.zone, "zone", "", "For use with gcloud commands to specify the cluster zone.")
+	flags.StringSliceVar(&d.regions, "region", []string{}, "Comma separated list for use with gcloud commands to specify the cluster region(s). The first region will be considered the primary region, and the rest will be considered the backup regions.")
+	flags.StringSliceVar(&d.zones, "zone", []string{}, "Comma separated list for use with gcloud commands to specify the cluster zone(s). The first zone will be considered the primary zone, and the rest will be considered the backup zones.")
+	flags.StringSliceVar(&d.retryableErrorPatterns, "retryable-error-patterns", []string{gceStockoutErrorPattern}, "Comma separated list of regex match patterns for retryable errors during cluster creation.")
```

Comment:
were you able to test if this worked by any chance? probably can have it such that it retries on all errors and specify an invalid version so we know it fails to test retying.

Reply:
We haven't tested it out in this specific context yet. However the error checking code is mostly copied from another PR that Chi wrote and has been validated there.

Reply:
It might be worth just testing out if the retry logic is working.